tools/api-markdown-documenter/pnpm-lock.yaml

```diff
@@ -892,7 +892,7 @@ packages:
   /@rushstack/rig-package/0.5.1:
```

Comment:
We should probably revert the changes to this file, since we didn't end up making any changes in this package.

Reply:
same with the root pnpm-lock

Reply:
Lock files reverted. However, I believe the change in `api-markdown-documenter` is necessary. We could talk about this during the parking lot.